code/testsuite/csheets/35e_Alice.xml

```diff
@@ -125,7 +125,7 @@ BIO
 		</reach>
 		<region>None</region>
 		<reputation>0</reputation>
-		<residence>Peachmond</residence>
+		<residence></residence>
```

Comment:
Trivial change, but why?

Reply:
Technically, it's listed as Alice's City, not Residence, so it's correct in being empty.  The XML file doesn't list city, so it's never shown there (we should probably update that XML template)